lua/wire/client/texteditor.lua

```diff
@@ -3109,6 +3109,7 @@ do -- E2 Syntax highlighting
 		["@inputs"] = 1, -- directive yellow, types orange, rest normal
 		["@outputs"] = 1,
 		["@persist"] = 1,
+		["@rawinputs"] = 1,
```

Comment:
Sorry, I missed it. Do you want me to add it or do you want to do it by yourself?